pallets/file-system/src/lib.rs

```diff
@@ -129,23 +168,17 @@ pub mod pallet {
 
     /// A pointer to the earliest available block to insert a new storage request expiration.
     ///
-    /// This should always be equal or greater than `current_block` + [`Config::StorageRequestTtl`].
-    ///
-    /// In the event when this value is smaller than `current_block` + `StorageRequestTtl` value, the
-    /// storage request expiration will be inserted in the block `StorageRequestTtl` ahead, and then
-    /// this value will be reset to block number a `current_block` + `StorageRequestTtl`.
+    /// This should always be greater or equal than current block + [`Config::StorageRequestTtl`].
```

Comment:
I think it is more bloat than valuable documentation. Also I don’t want to explain the algorithm here and in `on_idle` since if something changes we have to remember to change it here as well.